.github/workflows/ct_reusable_monitoring.yml

```diff
@@ -74,7 +78,13 @@ jobs:
         run: cat ${{ env.LOG_FILE }}
         # Skip on first run
         continue-on-error: true
-      - run: go run ./cmd/ct_monitor --config ${{ inputs.config }} --file ${{ env.LOG_FILE }} --once=${{ inputs.once }}
+      - run: |
+          go run ./cmd/ct_monitor \
+          --config ${{ inputs.config }} \
+          --file ${{ env.LOG_FILE }} \
+          --once=${{ inputs.once }} \
+          --url ${{ inputs.url }}
```

Comment:
Can we update this to be the same as the other command with the optional `--url` param?